packages/paste-core/utilities/grid/__tests__/grid.test.tsx

```diff
@@ -0,0 +1,588 @@
+      .create(
+        <Theme.Provider theme="console">
+          <Grid>
+            <Column span={[2, 4, 2]} offset={[8, 6, 8]}>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+            <Column>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+          </Grid>
+        </Theme.Provider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
+
+describe('Column Span Prop', () => {
+  it('it should not set a span property', (): void => {
+    const tree = renderer
+      .create(
+        <Theme.Provider theme="console">
+          <Grid>
+            <Column>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+            <Column>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+          </Grid>
+        </Theme.Provider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('it should set a span property', (): void => {
+    const tree = renderer
+      .create(
+        <Theme.Provider theme="console">
+          <Grid>
+            <Column span={8}>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+            <Column span={4}>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+          </Grid>
+        </Theme.Provider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('it should set a responsive span property', (): void => {
+    const tree = renderer
+      .create(
+        <Theme.Provider theme="console">
+          <Grid>
+            <Column span={[8, 4, 2]}>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+            <Column span={[4, 8, 10]}>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+          </Grid>
+        </Theme.Provider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
+
+describe('12 Column Options', () => {
+  it('it should render a 12 column grid', (): void => {
+    const tree = renderer
+      .create(
+        <Theme.Provider theme="console">
+          <Grid>
+            <Column>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+            <Column>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+            <Column>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+            <Column>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+            <Column>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
```

Comment:
Can we omit these child `Box`s from our tests?  The only functionality we're testing is the `Grid` and `Column` components really.  An empty div would work if a child is required `<div />`